shared/src/question-generators/example/example.ts

```diff
@@ -14,8 +30,12 @@ import { t, tFunctional, Translations } from "../../utils/translations"
 const translations: Translations = {
   en: {
     name: "Compute a sum",
-    description: "Compute the sum of two integers",
-    text: "Let ${{0}}$ and ${{1}}$ be two natural numbers. What is the **sum** ${{0}}+{{1}}$?",
+    description: `Compute the sum of two integers`,
+    text: `Let {{0}} and {{1}} be two - natural numbers. What is the **sum** {{0}}+{{1}}?
+\`\`\`pseudoCode
+{{2}}
+\`\`\`
```

Comment:
maybe we want to have a function that turns a pseudocode object into such a codeblock with the right name and stringifies the objects. This will be the frequent pattern. And if we want to change it later we only need to change the function.

Reply:
Sounds good, I'll add it. But I would still just use `JSON.stringify` to convert the PseudoCode Object into a Json string, right?

Reply:
yes, but i would do it in that function. So the markdown in the question would just look like `{{1}}` and the code for the data to insert here is `stringifyPseudoCode(pseudoCode)` or something like that.